.changeset/three-walls-lay.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+`Table`: Removed unused updateAriaLabel function and event listener
```

Comment:
sorry, we [settled on using dash separators](https://hashicorp.atlassian.net/wiki/spaces/HDS/pages/3243114706/Changelog+authoring+best+practices#Component-changes); happy for us to challenge this, but in the meantime we'd have to adhere to the format
```suggestion
`Table` – Removed unused `updateAriaLabel` function and event listener
```